cmake_files/dependencies.cmake

```diff
@@ -49,3 +49,7 @@ if(dompi)
   endif()
 endif()
 set(SOPT_MPI ${MPI_FOUND})
+
+if(cppflow)
```

Comment:
@tkoskela we wanted to use `if(TENSORFLOW_LIB_FOUND)` here rather than `if(cppflow)` but that variable isn't created by the `find_library` function, only by `find_package`. The [docs for find_library](https://cmake.org/cmake/help/latest/command/find_library.html) say

> If the library is found the result is stored in the variable and the search will not be repeated unless the variable is cleared. If nothing is found, the result will be <VAR>-NOTFOUND.

But I'm not sure how to check for that here. Would it be something like something like:
`if(TENSORFLOW_LIB != "TENSORFLOW_LIB-NOTFOUND"` ??